token/js/test/e2e-2022/groupMember.test.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from 'chai';
+import type { Connection, Signer } from '@solana/web3.js';
+import { PublicKey } from '@solana/web3.js';
+import { sendAndConfirmTransaction, Keypair, SystemProgram, Transaction } from '@solana/web3.js';
+
+import {
+    ExtensionType,
+    createInitializeMintInstruction,
+    getTokenGroupState,
+    getMint,
+    getMintLen,
+    createInitializeGroupMemberPointerInstruction,
+    createInitializeGroupPointerInstruction,
+    getTokenGroupMemberState,
+    tokenGroupInitializeGroupWithRentTransfer,
+    tokenGroupMemberInitializeWithRentTransfer,
+} from '../../src';
+import { TEST_PROGRAM_ID, newAccountWithLamports, getConnection } from '../common';
+
+const TEST_TOKEN_DECIMALS = 2;
+const EXTENSIONS = [ExtensionType.GroupMemberPointer];
+
+describe('tokenGroupMember', async () => {
+    let connection: Connection;
+    let payer: Signer;
+    let mint: Keypair;
+    let mintAuthority: Keypair;
+    let updateAuthority: Keypair;
+    let groupAddress: PublicKey;
+    let memberAddress: PublicKey;
+
+    before(async () => {
+        connection = await getConnection();
+        payer = await newAccountWithLamports(connection, 1000000000);
+        mintAuthority = Keypair.generate();
+        updateAuthority = Keypair.generate();
+    });
+
+    beforeEach(async () => {
+        mint = Keypair.generate();
+        groupAddress = PublicKey.unique();
+        memberAddress = PublicKey.unique();
+
+        const mintLen = getMintLen(EXTENSIONS);
+        const lamports = await connection.getMinimumBalanceForRentExemption(mintLen);
+
+        const transaction = new Transaction().add(
+            SystemProgram.createAccount({
+                fromPubkey: payer.publicKey,
+                newAccountPubkey: mint.publicKey,
+                space: mintLen,
+                lamports,
+                programId: TEST_PROGRAM_ID,
+            }),
+            createInitializeGroupMemberPointerInstruction(
+                mint.publicKey,
+                mintAuthority.publicKey,
+                memberAddress,
+                TEST_PROGRAM_ID
+            ),
+            createInitializeMintInstruction(
+                mint.publicKey,
+                TEST_TOKEN_DECIMALS,
+                mintAuthority.publicKey,
+                null,
+                TEST_PROGRAM_ID
+            )
+        );
+
+        await sendAndConfirmTransaction(connection, transaction, [payer, mint], undefined);
+    });
+
+    it('can initialize group member', async () => {
+        const tokenGroupMember = {
+            mint: mint.publicKey,
+            group: groupAddress,
+            memberNumber: 1,
+        };
+
+        await tokenGroupMemberInitializeWithRentTransfer(
+            connection,
+            payer,
+            memberAddress,
+            mint.publicKey,
+            mintAuthority.publicKey,
+            groupAddress,
+            updateAuthority.publicKey,
```

Comment:
No sure why this test fails with `Missing signature for public key ` for `updateAuthority`

Reply:
Yeah both the member's mint authority and the group's update authority have to sign here.

https://github.com/solana-labs/solana-program-library/blob/e651623033fca7997ccd21e55d0f2388473122f9/token-group/js/src/instruction.ts#L128

You also need to create the group mint first.